index.html

```diff
@@ -97,6 +97,33 @@
       .usa-stripe:nth-child(odd){
         background-color: #FFFFFF;
       }
+      #ireland{
+        height: 61vw;
+        grid-template-columns: 1fr 1fr 1fr;
+      }
+      .ireland-stripe:nth-child(1){
+        background-color:#4C9C62;
+      }
+      .ireland-stripe:nth-child(2){
+        background-color:#FFFFFF;
+      }
+      .ireland-stripe:nth-child(3){
+        background-color:#E9863F;
+      }
+      #sweden{
+        height: 61vw;
+        grid-template-columns: 5fr 2fr 11.2fr;
+        grid-template-rows: 4fr 2fr 4fr;
+      }
+      .swed-bl{
+        background-color: #3668A1;
+      }
+      .swed-yl{
+        background-color: #ECC505;
+      }
+      .stripe-full{
+        grid-column: 1 / -1;
+      }
```

Comment:
Can we move this declaration to the top of the styling -- right under `.flag` -- since it's used in many different flags?